devel/gl-headers/Portfile

```diff
@@ -7,9 +7,10 @@ PortGroup           meson 1.0
 gitlab.setup        meson-ports gl-headers 5c8c7c0d
 gitlab.instance     https://gitlab.freedesktop.org/gstreamer
 version             2019.1.0
-revision            0
+revision            1
```

Comment:
Does it really need a revbump?

Reply:
As far as I know, the bundled Portfile is only used activate and deactivate steps, in case the Portfile has custom pre- or post- -activate or -deactivate blocks.